src/components/workspace-utils.js

```diff
@@ -97,7 +97,11 @@ export const withWorkspaces = (WrappedComponent) => {
 export const WorkspaceSelector = ({ workspaces, value, onChange, id, 'aria-label': ariaLabel, ...props }) => {
   const options = _.flow(
     _.sortBy((ws) => ws.workspace.name.toLowerCase()),
-    _.map(({ workspace: { workspaceId, name } }) => ({ value: workspaceId, label: name }))
+    _.map(({ workspace: { workspaceId, name, cloudPlatform, bucketName } }) => ({
+      value: workspaceId,
+      label: name,
+      workspace: { cloudPlatform, bucketName },
```

Comment:
Is there a downside to passing the entire `workspace` object here, in case other fields are needed in the future?
```suggestion
    _.map((workspace) => ({
      value: workspace.workspaceId,
      label: workspace.name,
      workspace,
```

Reply:
Hmm, I guess that's cleaner than what I did, and I'm not sure what the downside would be.